src/components/Checkout/AddressForm.test.js

```diff
@@ -0,0 +1,55 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import AddressForm from "./AddressForm";
+
+describe("<AddressForm/>", () => {
+  describe("when Address Form is rendered", () => {
+    it("renders the addresses fields", () => {
+      render(<AddressForm />);
+      const zipCode = screen.getByRole('textbox', {  name: /zip \/ postal code/i});
+      expect(zipCode).toBeInTheDocument();
+      const city = screen.getByRole('textbox', {  name: /city/i})
+      expect(city).toBeInTheDocument();
+      const country = screen.getByRole('button', {  name: /country ​/i});
+      expect(country).toBeInTheDocument();
+      const region = screen.getByRole('button', {  name: /state\/province\/region ​/i});
+      expect(region).toBeInTheDocument();
+    });
+  });
+
+  describe("when renders the country field", () => {
+    it('displays the correct number of options', () => {
+      render(<AddressForm />)
+      const country = screen.getByRole('button', {  name: /country ​/i});
+      userEvent.click(country);
+      const numberOfOptions = screen.getAllByRole('option');
+      expect(numberOfOptions.length).toBe(191)
```

Comment:
The number `191` here seems mysterious. Can we assign it to a constant and use the constant here?